sn_networking/src/driver.rs

```diff
@@ -608,6 +615,21 @@ impl SwarmDriver {
                         bootstrap_interval = new_interval;
                     }
                 }
+                _ = set_farthest_record_interval.tick() => {
+                    let closest_peers = self
+                    .swarm
+                    .behaviour_mut()
+                    .kademlia
+                    .get_closest_local_peers(&self.self_peer_id.into())
+                    .map(|peer| peer.into_preimage())
+                    .take(CLOSE_GROUP_SIZE)
```

Comment:
here passed CLOSE_GROUP_SIZE peers in,
but the `is_in_close_range` function requires the input list to be greater than REPLICATE_RANGE,
which actually will always return true ?